README.md

```diff
@@ -769,49 +776,51 @@ subscriptions example:
 Process Subscriptions:
 process.target = "https://mysite.com/beehire/notification/"
 process.signed = true
-process.events = "error,success,deliver,present,download,delete,reroute"
+process.events = ["error", "success", "deliver", "present", "download", "delete", "reroute"]
 
 Bee subscriptions:
 xcoobee_testbee.target = "https://somesite.com/testbee/notification/"
-xcoobee_testbee.events = "success,error"
```

Comment:
Vitaliy, do we remove the subscription object.
Bee subscriptions are specific to each transaction. Users have to subscribe with each hire. Are we making this generic regardless of transaction and hire?